microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools.md

```diff
@@ -0,0 +1,288 @@
+title: What's new in DevTools (Microsoft Edge 93)
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 07/30/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+# What's New In DevTools (Microsoft Edge 93)
+
+[!INCLUDE [contact DevTools team note](../../includes/edge-whats-new-note.md)]
+
+<!-- 1 -->
+## Apply themes from Visual Studio Code to DevTools
+
+<!-- Title: Apply themes from Visual Studio Code to DevTools -->
+<!-- Subtitle: You can now use some of the most popular color themes from Visual Studio Code, such as Monokai and Solarized Dark, in Microsoft Edge DevTools. -->
+
+In addition to the existing light and dark themes, Microsoft Edge DevTools now supports some of the most popular color themes from Visual Studio Code.  To select a color theme, navigate to **Settings** and then select a theme from the **Theme** dropdown list.
+
+:::image type="complex" source="../../media/2021/07/all-devtools-themes.msft.png" alt-text="Color themes for DevTools" lightbox="../../media/2021/07/all-devtools-themes.msft.png":::
+   Color themes for DevTools
+:::image-end:::
+
+The supported Visual Studio Code themes are:
+
+Light themes:
+*  Solarized Light
+*  Quiet Light
+
+Dark themes:
+*  Abyss
+*  Kimbie Dark
+*  Monokai
+*  Monokai Dimmed
+*  Solarized Dark
+*  Red
+*  Tomorrow Night Blue
+
+For more information, navigate to [Apply color themes to Microsoft Edge DevTools](../../../customize/dark-theme.md).
+
+<!-- no crbug -->
+
+
+<!-- 2 -->
+## Debug DOM node memory leaks with the new Detached Elements tool
+
+:::image type="icon" source="../../media/2020/06/experimental-tag-14px.msft.png":::  
+
+<!-- Title: Introducing the Detached Elements tool -->
+<!-- Subtitle: Use the Detached Elements tool to find and fix DOM node memory leaks. -->
+
+A DOM node is considered "detached" when it is no longer attached to any element of the DOM but is still being retained in memory by Microsoft Edge. The browser cannot garbage-collect the element, because some JavaScript is still referencing the element even though it's no longer on the page or is not longer a part of the DOM.
+
+The new **Detached Elements** tool finds all of the detached elements on your page and displays them. You can expand and collapse a detached element to see the parent and child nodes that are also being retained. You can trigger the browser's garbage collection by selecting **Collect garbage** and validate that you have a memory leak when a detached element cannot be garbage-collected. And you can jump into JavaScript that's referencing the detached element by taking a heap snapshot with the **Analyze** button.
+
+<!-- todo: comment-out or delete steps? -->
+1. In Microsoft Edge Canary, navigate to [Microsoft ToDo](https://to-do.office.com/tasks/inbox).
```

Comment:
It wasn't clear to me at first that the numbered steps were for a tutorial. Maybe it would be helpful to add a sentence before the list that says something like "A good way to test the Detached Elements tool is by debugging a to-do list web application such as Microsoft To Do." That way, it's clear that the steps are a tutorial and not required setup instructions.